Dshop/apps/products_catalogue/templates/products_catalogue/product_detail.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+
+{% load thumbnail %}
+
+{% block content %}
+
+<div class="container mt-5 mb-5">
+    <div class="row d-flex justify-content-center">
+        <div class="col-md-10">
+            <div class="card">
+                <div class="row">
+                    <div class="col-md-6">
+                        <div class="images p-3">
+                            {% for pic in product.images.all %}
+                                {% if pic.is_featured %}
```

Comment:
please remove the if, since it's details page, it should show all photos.

Reply:
True, my mistake